Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -72,7 +72,7 @@ FORCE_INLINE void _draw_centered_temp(const int16_t temp, const uint8_t tx, cons
 
 #define DO_DRAW_BED (HAS_HEATED_BED && STATUS_BED_WIDTH && HOTENDS <= 3 && DISABLED(STATUS_COMBINE_HEATERS))
 #define DO_DRAW_FAN (HAS_FAN0 && STATUS_FAN_WIDTH && STATUS_FAN_FRAMES)
-//#define DO_DRAW_CHAMBER (HAS_TEMP_CHAMBER && ((HOTENDS <= 2 && DO_DRAW_BED) || (!DO_DRAW_BED && HOTENDS <= 3)))
+#define DO_DRAW_CHAMBER (HAS_TEMP_CHAMBER && ((HOTENDS <= 2 && DO_DRAW_BED) || (!DO_DRAW_BED && HOTENDS <= 3)))
```

Comment:
I just saw now a define that may help:
#define DO_DRAW_CHAMBER (HAS_TEMP_CHAMBER && (HOTENDS == 1 || HOTENDS < MAX_HOTEND_BITMAPS))

Reply:
Follow the bed model as closely as possible. Note especially that the chamber should not be drawn if `ENABLED(STATUS_COMBINE_HEATERS)`, because in that case the chamber will be included in the single header graphic already.

Reply:
Doing that in following commit